### PR TITLE
Extend package info on uninstall operation

### PR DESCRIPTION
**Pull request by YuliiaKovalova**
### Problem
It was agreed to extend package info on `--uninstall` command with information about Owners and trustiness identifier.

### Solution
The information is provided by nuget api

### Checks:
- [ n/a ] Added unit tests
- [ n/a ] Added `#nullable enable` to all the modified files [?](https://github.com/dotnet/templating/wiki/Contributing#coding-style)

### CODE REVIEW

**Review by vlada-shubina — COMMENTED** [review]
Please extend tests in https://github.com/dotnet/templating/blob/main/test/Microsoft.TemplateEngine.Edge.UnitTests/NuGetInstallerTests.cs and https://github.com/dotnet/templating/blob/main/test/Microsoft.TemplateEngine.Edge.UnitTests/NuGetApiPackageManagerTests.cs

File: src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs [hunk 1]
```diff
@@ -391,6 +398,8 @@ private NuGetPackageInfo ReadPackageInformation(string packageLocation)
 
             return new NuGetPackageInfo(
                 nuspec.GetAuthors(),
+                nuspec.GetOwners(),
+                PrefixReserved,
```
Comment (vlada-shubina) [inline src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs:402]: nit: we may just return `false` here, property on `NuGetInstaller` level is a bit confusing.

**Comment by YuliiaKovalova** [comment]
> Please extend tests in https://github.com/dotnet/templating/blob/main/test/Microsoft.TemplateEngine.Edge.UnitTests/NuGetInstallerTests.cs and https://github.com/dotnet/templating/blob/main/test/Microsoft.TemplateEngine.Edge.UnitTests/NuGetApiPackageManagerTests.cs

done

File: src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NugetPackageMetadata.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using NuGet.Packaging.Core;
+using NuGet.Protocol.Core.Types;
+
+namespace Microsoft.TemplateEngine.Edge.Installers.NuGet
+{
+    internal class NugetPackageMetadata
```
Comment (vlada-shubina) [inline src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NugetPackageMetadata.cs:9]: may be private to `NugetApiPackageManager`

File: src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetManagedTemplatePackage.cs [hunk 3]
```diff
@@ -81,6 +83,23 @@ internal class NuGetManagedTemplatePackage : IManagedTemplatePackage
...
+
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value))
+                {
+                    Details.Remove(VerifiedKey);
+                }
+                else
+                {
+                    Details[VerifiedKey] = bool.TryParse(value, out bool verified) && verified ? "✔" : "✘";
```
Comment (vlada-shubina) [inline src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetManagedTemplatePackage.cs:98]: please consider keeping it as true or false and leave final representation to the host.
For example, VS might want to use different representation and custom symbols are hard to parse.

**Pull request merged**